resources/bundle-extends/test.js

```diff
@@ -0,0 +1,14 @@
+import { bundleConfig } from '../../packages/core/lib/bundle.js';
+import { BaseResolver } from '../../packages/core/lib/resolve.js';
+import { resolveConfigFileAndRefs } from '../../packages/core/lib/config/config-resolvers.js';
```

Comment:
There's no `resolveConfigFileAndRefs` anymore. Remove this file?

Reply:
removed